<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SummerProject</class>
 <widget class="QMainWindow" name="SummerProject">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777212</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Учитель</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Custom_View" name="image_viewer">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>630</width>
      <height>540</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="0">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="foregroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="0">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QPushButton" name="upload_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 LtEx BT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Upload image</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="draw_rectangle">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>12</y>
      <width>102</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 LtEx BT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rectangle</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="draw_line">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>12</y>
      <width>58</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 LtEx BT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearAll_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 LtEx BT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Clear all</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="draw_highlighter">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>356</x>
      <y>12</y>
      <width>86</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 LtEx BT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Highliter</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hand">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>448</x>
      <y>12</y>
      <width>67</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 LtEx BT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hand</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Custom_View</class>
   <extends>QGraphicsView</extends>
   <header location="global">custom_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
